src/Enterspeed.Source.UmbracoCms.V8/Services/UmbracoRichTextParser.cs

```diff
@@ -46,6 +47,13 @@ public string PrefixRelativeImagesWithDomain(string html, string mediaDomain)
                 var src = imageNode.GetAttributeValue("src", string.Empty);
                 if (src.StartsWith("/media/"))
                 {
+                    var absolutePath = mediaDomainUrl.AbsolutePath;
+                    if (!absolutePath.Equals("/"))
+                    {
+                        src = absolutePath + src;
```

Comment:
We dont have path.join in classic .NET Framework. Combine does not give us the desired result. Do you think this would be an ok approach?

Reply:
Personally I think it is a fine approach, if we don't have anything else available.

Reply:
I think Path.Join will add a **directory** separator (`\\`) between each part if a separator is missing. so e.g. `Path.Join("path1", "path2")` becomes `path1\\path2` and not `path1/path2`.